team/README.md

```diff
@@ -1,16 +1,20 @@
 # The team folder
 
-All members of the organizing team should add themselves to the webpage and
-this folder contains a file for each person describing themselves.
+All members of the organizing team should add themselves to the webpage.
+This folder contains a file for each person describing themselves.
 
 ## Adding a new member
 Follow these steps to add yourself:
-* Use the provided 'template.yaml' file and copy the file.
-* Create a copy of the 'template.yaml' file and rename the file to your name
+* Navigate to the 'landing page' repository for your event.
+* Navigate into the 'teams' directory.
+* Use the provided 'template.yaml' file.
+* Create a copy of the 'template.yaml' file and rename the file to your name.
+You can do this through the GitHub interface, your local development environment, or on the command line with
```

Comment:
Curious - How does one copy a file within the GitHub UI?

Reply:
You can't, directly. You'd have to create a new file and copy the content...

Reply:
Maybe we should simplify this second sentence then. I also don't see much of a difference between `local dev environment` and the `command line`.
Or should we just skip saying you should do this via the GitHub UI and only recommend using a local cloned repo?